code/datums/components/infective.dm

```diff
@@ -34,6 +37,11 @@
 		eater.ForceContractDisease(V)
 	try_infect(feeder, BODY_ZONE_L_ARM)
 
+/datum/component/infective/proc/try_infect_drink(datum/source, mob/living/drinker, mob/living/feeder)
+	for(var/V in diseases)
+		drinker.ForceContractDisease(V)
+	try_infect(feeder, BODY_ZONE_L_ARM)
```

Comment:
Why the left arm?

Reply:
No specific reason, the rest of the file used the left arm while infecting too and I thought to keep it uniform